core/src/main/resources/hudson/logging/LogRecorderManager/levels.jelly

```diff
@@ -57,22 +57,27 @@ THE SOFTWARE.
     <l:isAdmin>
       <h2>${%Adjust Levels}</h2>
       <form action="configLogger" method="post" name="configLogger">
-        ${%Name}: <input type="text" name="name" width="40" />
-        ${%Level}:
-        <select name="level">
-          <option>inherit</option>
-          <option>off</option>
-          <option>severe</option>
-          <option>warning</option>
-          <option>info</option>
-          <option>config</option>
-          <option>fine</option>
-          <option>finer</option>
-          <option>finest</option>
-          <option>all</option>
-        </select>
-        <st:nbsp/>
-        <f:submit />
+        <div class="jenkins-form-item">
```

Comment:
Would be nice to have each input + label wrapped in its own `jenkins-form-item` or `<f:entry/>` as this would fix the spacing between the components.